src/core/server/saved_objects/migrations/kibana/kibana_migrator.ts

```diff
@@ -120,61 +151,22 @@ export class KibanaMigrator {
       });
     });
 
-    if (migrators.length === 0) {
```

Comment:
migrators.length would be 0 if there are no `savedObjectMappings` defined. This could happen if there are no plugins enabled or if none of the plugins use saved objects. This shouldn't normally happen, but if it does it is a valid way to run Kibana if you want.